anomalib/models/components/sampling/k_center_greedy.py

```diff
@@ -11,7 +11,7 @@
 import torch.nn.functional as F
 from torch import Tensor
 
-from anomalib.core.model.random_projection import SparseRandomProjection
+from anomalib.models.components.dimensionality_reduction import SparseRandomProjection
```

Comment:
We could import this via `from anomalib.models.components import SparseRandomProjection`, right?